v0.1/core/brainAtlasVersion.schema.json

```diff
@@ -0,0 +1,86 @@
+{
+  "$id": "https://schema.hbp.eu/sand/0.1/core/brainAtlas.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "Definition of a brain atlas",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "@type",
+    "@id",
+    "alias",
+    "annotationSet",
+    "fullName",
+    "releaseDate",
+    "terminology",
+    "versionIdentifier",
+    "versionInnovation" 
+  ],
```

Comment:
According to issue #29, `coordinateSpace` is also required.

Reply:
yes. I forgot that one. I'll add it.